lib/ts/recipe/passwordless/api/implementation.ts

```diff
@@ -47,7 +47,14 @@ export default function getAPIImplementation(): APIInterface {
                 }
             }
 
-            const session = await Session.createNewSession(input.options.res, user.id, user.recipeUserId, {}, {}, input.userContext);
+            const session = await Session.createNewSession(
```

Comment:
email verification related functions everywhere should use recipeUserId instead user.id (see above code in this file)

Reply:
This can be done as a different PR once this PR is merged into the base branch